pages/blog/css-with-react.mdx

```diff
@@ -0,0 +1,274 @@
+---
+title: 'How Should I Style My React Application?'
+publishedAt: '2020-09-15'
+summary: 'TODO'
+image: '/static/images/css-with-react/banner.png'
+---
+
+The React ecosystem is unopinionated. **This is both it's greatest strength and weakness.**
```

Comment:
I am not a native speaker but I think that the ecosystem can be opinionated – React however is not a full blown framework and therefore doesn't prescribe how to do many things including styling.

Reply:
That's a good clarification! Let me tweak this.